Makefile

```diff
@@ -6,3 +6,6 @@ build: rock
 
 rock:
 	luarocks make wildcard-service-scm-1.rockspec
+
+prove: 
+	TEST_NGINX_ERROR_LOG=/dev/stderr TEST_NGINX_BINARY=openresty prove
```

Comment:
we might want to set `TEST_NGINX_BINARY=openresty-debug` on CircleCI so it prints debug log.

That can be done by variables with defaults in the Makefile.